chain-impl-mockchain/src/vote/manager.rs

```diff
@@ -347,7 +347,7 @@ impl ProposalManager {
             result.add_vote(Choice::new(u8::try_from(choice).unwrap()), weight)?;
         }
 
-        if self.check((*total_stake).into(), governance, &result) {
+        if self.check(result.participation(), governance, &result) {
```

Comment:
Sorry if I was misleading, I meant to say we should use something like `token_distribution.get_total()`, which btw is what we do on master. Rebasing should fix this